src/dotnet-scaffolding/dotnet-scaffold-aspnet/Helpers/ViewHelper.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.TextTemplating;
+using Microsoft.DotNet.Tools.Scaffold.AspNet.Models;
+using Microsoft.DotNet.Tools.Scaffold.AspNet.Templates.Views;
+
+namespace Microsoft.DotNet.Tools.Scaffold.AspNet.Helpers;
+
+internal class ViewHelper
+{
+    internal const string CreateTemplate = "Create.tt";
+    internal const string DeleteTemplate = "Delete.tt";
+    internal const string DetailsTemplate = "Details.tt";
+    internal const string EditTemplate = "Edit.tt";
+    internal const string IndexTemplate = "Index.tt";
+
+    internal static IEnumerable<TextTemplatingProperty> GetTextTemplatingProperties(IEnumerable<string> allT4TemplatePaths, ViewModel viewModel)
+    {
+        var textTemplatingProperties = new List<TextTemplatingProperty>();
+        foreach (var templatePath in allT4TemplatePaths)
```

Comment:
If `allT4TemplatePaths` is `null`, this line will throw

Reply:
fixed, with nullability, it would be a warning and which is treated as an error in this repo, but better to be defensive and will serve me better when writing external APIs.